fhir-server/src/main/java/com/ibm/fhir/server/util/RestAuditLogger.java

```diff
@@ -59,411 +58,521 @@
 
     private static final String COMPONENT_ID = "fhir-server";
 
+    private static final ComponentIpHandler componentIpHandler = new ComponentIpHandler();
 
     /**
      * Builds an audit log entry for a 'create' REST service invocation.
-     * @param request - The HttpServletRequest representation of the REST request.
-     * @param resource - The Resource object being created.
-     * @param startTime - The start time of the create request execution.
-     * @param endTime - The end time of the create request execution.
-     * @param responseStatus - The response status.
+     *
+     * @param request
+     *            - The HttpServletRequest representation of the REST request.
```

Comment:
Minor style comment:  I don't think we use the `-` in other places where we add the param description on a new line.
```suggestion
     *            The HttpServletRequest representation of the REST request.
```
This applies to all the other places where you've reformatted the javadoc in this class as well.

Reply:
I think it's artifact of the class author.